bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/ESH-INF/thing/airthings.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="airthings_wave_plus">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bluegiga" />
+			<bridge-type-ref id="bluez" />
+		</supported-bridge-type-refs>
+
+		<label>Airthings Wave Plus</label>
+		<description>Indoor air quality monitor with radon detection</description>
+
+		<channels>
+			<channel id="rssi" typeId="rssi" />
+
+			<channel id="humidity" typeId="airthings_humidity" />
+			<channel id="temperature" typeId="airthings_temperature" />
+			<channel id="pressure" typeId="airthings_pressure" />
+			<channel id="co2" typeId="airthings_co2" />
+			<channel id="tvoc" typeId="airthings_tvoc" />
+			<channel id="radon_st_avg" typeId="airthings_radon_st_avg" />
+			<channel id="radon_lt_avg" typeId="airthings_radon_lt_avg" />
+		</channels>
+
+		<config-description>
+			<parameter name="address" type="text">
+				<label>Address</label>
+				<description>Bluetooth address in XX:XX:XX:XX:XX:XX format</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer">
+				<label>Refresh Interval</label>
+				<description>States how often a refresh shall occur in seconds. This could have impact to battery lifetime</description>
+				<default>300</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="airthings_humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Humidity level (%%)</description>
```

Comment:
Applies to all channels: Since you use UoM, adding the unit in the description is not needed (and probably wrong if the user selected a different unit).